client/components/domains/registrant-extra-info/uk-form.jsx

```diff
@@ -137,12 +164,31 @@ class RegistrantExtraInfoUkForm extends React.PureComponent {
 						autoCorrect="off"
 						placeholder={ '' }
 						onChange={ this.handleChangeEvent }
+						isError={ isError }
 					/>
+					{ map( registrationNumberErrors, this.renderValidationError ) }
 				</FormFieldset>
 			</div>
 		);
 	}
 
+	errorMessageFromCode( errorCode ) {
+		return (
+			this.errorMessages[ errorCode ] ||
+			this.props.translate( 'There was a problem with this field.' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'There was a problem with this field' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).